app/assets/javascripts/topbar.js

```diff
@@ -69,21 +52,15 @@ function is_mobile() {
   return check;
 }
 
-function onScroll(item){
-  var offset = 25;
-  var dist = $(item).scrollTop();
-  var header_height = $('.persist-header:first').height();
+function persistHeader(){
+  var offset = $('.navbar-outer').height();
```

Comment:
let the cutoff location be determined by the top navbar height instead of hard coded - in case its height changes.